tendrl/ceph_integration/objects/pool/atoms/rename/__init__.py

```diff
@@ -0,0 +1,71 @@
+from tendrl.ceph_integration.manager.crud import Crud
+from tendrl.ceph_integration.manager.exceptions import \
+    RequestStateError
+from tendrl.commons import objects
+from tendrl.ceph_integration.objects.pool import Pool
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+from tendrl.commons.objects import AtomExecutionFailedError
+
+
+class Rename(objects.BaseAtom):
+    obj = Pool
+    def __init__(self, *args, **kwargs):
+        super(Rename, self).__init__(*args, **kwargs)
+
+    def run(self):
+        pool_id = self.parameters['Pool.pool_id']
+        attrs = {}
+        attrs['name'] = self.parameters.get('Pool.poolname')
+        Event(
+            Message(
+                priority="info",
+                publisher=NS.publisher_id,
+                payload={
+                    "message": "Renaming the pool with pool-id %s."
+                    " New name: %s" %
+                    (self.parameters['Pool.pool_id'],
+                     self.parameters.get('Pool.poolname'))
+                    },
+                job_id=self.parameters['job_id'],
+                flow_id=self.parameters['flow_id'],
+                cluster_id=NS.tendrl_context.integration_id,
+            )
+        )
+
+        crud = Crud()
+        resp = crud.update("pool", pool_id, attrs)
+        try:
+            crud.sync_request_status(resp['request'])
+        except RequestStateError as ex:
+            Event(
+                Message(
+                    priority="info",
+                    publisher=NS.publisher_id,
+                    payload={
+                        "message": "Failed to rename pool %s."
```

Comment:
For old name we would need to fetch the pool from etcd. Anyway will add new name at least.